test/mdm_test.cc

```diff
@@ -221,7 +233,6 @@ int main(int argc, char **argv) {
   TestRenameBucket(hermes);
   TestBucketRefCounting(hermes);
   TestMaxNameLength(hermes);
-  TestGetRelativeNodeId();
```

Comment:
Why is this test deleted?

Reply:
Should be a mistake. I will get it back.